client/src/stores/user.ts

```diff
@@ -0,0 +1,41 @@
+import { defineStore } from 'pinia'
+import { useStorage } from '@vueuse/core'
+import { computed } from 'vue'
+
+// User interface (include userName, name and email)
+interface User {
+  userName: string
+  name: string
+  email: string
+}
+
+export const useUserStore = defineStore('user', () => {
+  const userData = useStorage<User | null>('userData', null)
+  const accessToken = useStorage<string | null>('accessToken', null)
+  const refreshToken = useStorage<string | null>('refreshToken', null)
+
+  const isLoggedIn = computed(() => !!userData.value)
+
+  // Login Status
+  const login = (user: User, tokens: { accessToken: string; refreshToken: string }) => {
```

Comment:
The ``login()`` function should be something a little different. It should take in login credentials (username and password), and then send a post request to ``api/token/`` to receive the tokens, and then store these tokens in the store. After that a get request should be sent to receive the details of the current user (this endpoint hasn't been implemented yet, that's [issue 29](https://github.com/codersforcauses/bluecrew/issues/29)) and that also gets stored in the store.

However, this isn't part of the scope of this issue, so for now you can just remove the function.